spyderlib/config.py

```diff
@@ -523,13 +543,21 @@ def is_ubuntu():
               'editor/find next': "F3",
               'editor/find previous': "Shift+F3",
               'editor/replace text': "Ctrl+H",
+              'editor/undo': 'Ctrl+U',
+              'editor/redo': 'Ctrl+Y',
+              'editor/cut': 'Ctrl+X',
+              'editor/copy': 'Ctrl+C',
+              'editor/paste': 'Ctrl+V',
+              'editor/delete': 'Delete',
+              'editor/select all': "Ctrl+A",
               # -- In plugins/editor
               'editor/show/hide outline': "Ctrl+Alt+O",
               'editor/show/hide project explorer': "Ctrl+Alt+P",
               'editor/new file': "Ctrl+N",
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
-              'editor/save all': "Ctrl+Shift+S",

```

Comment:
Giving that you're changing this shortcut, you need to do a minor update in `CONF_VERSION`